.eslintrc

```diff
@@ -17,7 +17,8 @@
   },
 
   "parserOptions": {
-    "sourceType": "module"
+    "sourceType": "module",
+    "project": "./tsconfig.json"
```

Comment:
Why was this needed? I suppose eslint assumes this to be the default.

Reply:
Actually, in my computer, it fails to run without specifying the project. Let me show ya:

```text
➜  fluent-check git:(nullish-coalescing) ✗ npm run lint

> fluent-check@1.0.0 lint
> eslint --ext '.js,.ts' src test


Oops! Something went wrong! :(

ESLint: 7.21.0

Error: Error while loading rule '@typescript-eslint/strict-boolean-expressions': You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.
Occurred while linting /Users/bytter/Development/fluent-check/src/FluentCheck.ts
    at Object.getParserServices (/Users/bytter/Development/fluent-check/node_modules/@typescript-eslint/experimental-utils/dist/eslint-utils/getParserServices.js:23:15)
    at create (/Users/bytter/Development/fluent-check/node_modules/@typescript-eslint/eslint-plugin/dist/rules/strict-boolean-expressions.js:103:37)
    at Object.create (/Users/bytter/Development/fluent-check/node_modules/@typescript-eslint/experimental-utils/dist/eslint-utils/RuleCreator.js:13:24)
    at createRuleListeners (/Users/bytter/Development/fluent-check/node_modules/eslint/lib/linter/linter.js:761:21)
    at /Users/bytter/Development/fluent-check/node_modules/eslint/lib/linter/linter.js:931:31
    at Array.forEach (<anonymous>)
    at runRules (/Users/bytter/Development/fluent-check/node_modules/eslint/lib/linter/linter.js:876:34)
    at Linter._verifyWithoutProcessors (/Users/bytter/Development/fluent-check/node_modules/eslint/lib/linter/linter.js:1175:31)
    at Linter._verifyWithConfigArray (/Users/bytter/Development/fluent-check/node_modules/eslint/lib/linter/linter.js:1273:21)
    at Linter.verify (/Users/bytter/Development/fluent-check/node_modules/eslint/lib/linter/linter.js:1228:25)
```

Reply:
Uh, looks like this one requires extra stuff then 👍